Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs

```diff
@@ -297,12 +297,7 @@ private record struct GridDrawJob : IParallelRobustJob
 
         public void Execute(int index)
         {
-            var vert = Vertices[index];
-            var adjustedVert = Vector2.Transform(vert, Matrix);
-            adjustedVert = adjustedVert with { Y = -adjustedVert.Y };
-
-            var scaledVert = ScalePosition(adjustedVert, MinimapScale, MidPoint);
-            ScaledVertices[index] = scaledVert;
+            ScaledVertices[index] = Vector2.Transform(Vertices[index], Matrix);
```

Comment:
You'll probably want to bump the batch size to like 64  with reduced stuff happening.